### PR TITLE
add cobra and make flags constants

### PR DESCRIPTION
**Pull request by mdawn**
### [Update `tls-checker` to Truss Standards](https://www.pivotaltracker.com/story/show/173477160)

Does these things:
1) use spf13/cobra 🐍  to have subcommands
2) adds a Version subcommand
3) uses constants for flags. why? See: https://github.com/trussworks/health-checker/pull/1#discussion_r444478808

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.0.0
```
Comment (chrisgilmerproj) [inline go.mod:7]: Run `go mod tidy` and double check that there isn't also an indirect dependency that shows up.

**Comment by chrisgilmerproj** [comment]
This looks great! For testing (and probably this should go in the README.md) I generally uncomment these lines:

https://github.com/trussworks/tls-checker/blob/master/main.go#L336-L337

And then I run

```sh
go run . check --schemes https --hosts "www.truss.works" --log-level info --timeout 15m
```

You should see failures. It proves that it exits on the bad TLS versions that are accepted.

I see one more update to make to this PR and then I think its ready to merge!

File: go.mod (hunk 1 above)
Comment (mdawn) [reply go.mod:7]: ~There is! `github.com/inconshreveable/mousetrap v1.0.0 h1:Z8tu5sraLXCXIcARxBp/8cbvlwVa7Z1NHg9XEKhtSvM=`~

oh nvm that's just a checksum update

Comment (chrisgilmerproj) [reply go.mod:7]: well thanks for trying!

**Review by chrisgilmerproj — APPROVED** [review]
🚀

**Pull request merged**